ImportExtension/AddImportOperation.swift

```diff
@@ -120,9 +120,12 @@ class AddImportOperation {
         self.buffer.lines.insert(importString, at: line)
         
         //add a new selection. Bug fix for #7
-        let selectionPosition = XCSourceTextRange.init(start: XCSourceTextPosition.init(line: 0, column: 0), end: XCSourceTextPosition.init(line: 0, column: 0))
```

Comment:
That issue was fixed with Xcode 8.x, I forgot which specific version.

So it _should_ crash on earlier versions of Xcode. However, there shouldn’t be any reason why people are still on old versions of Xcode.